### PR TITLE
[FEAT] Revert Christmas Decorations

### PR DESCRIPTION
**Pull request by c-py**
# Description

This PR removes all christmas related decorations. Please click around and make sure nothing has been missed.

christmas-be-gone

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Visually tested

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [X] Screenshot if it includes any UI changes
- [X] I have read the contributing guidelines and agree to the T&Cs
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by spencerdezartsmith — DISMISSED** [review]
Grinch Grinch Grinch.... 🏖️ Back to the beach vibes

File: src/features/island/fruit/FruitPatch.tsx [hunk 1]
```diff
@@ -26,6 +25,46 @@ export const isReadyToHarvest = (
   );
 };
 
+import appleTree from "assets/fruit/apple/apple_tree.png";
```
Comment (spencerdezartsmith) [inline src/features/island/fruit/FruitPatch.tsx:28]: Can we please remove this file from the PR?

Comment (c-py) [reply src/features/island/fruit/FruitPatch.tsx:28]: aight

**Comment by mGranTY** [comment]
I hope it was easy to revert 😫

**Review by HBandesh — APPROVED** [review]
Great work!

**Pull request merged**